deploy/dev-ngsa-asb-westus2/burst/burst-flux-kustomization.yaml

```diff
@@ -10,7 +10,7 @@ spec:
     namespace: istio-system
   interval: 1m0s # detect drift and undo kubectl edits every minute
   prune: true # remove stale resources from cluster
-  targetNamespace: burstservice # overwrite the ns for resources in path
+  # targetNamespace: burstservice # overwrite the ns for resources in path
```

Comment:
Out of curiosity, how did this fix the issue?

Reply:
@AAkindele 
Burstmetrics service is deployed in `burstservice` namespace. But in burst directory there is an [envoy filter file](https://github.com/retaildevcrews/ngsa-asb/blob/main/deploy/dev-ngsa-asb-westus2/burst/remote-filter-ngsa-cosmos.yaml) which needed to be applied to `ngsa` namespace (for istio-sidecar-proxy).

Now, with `targetNamespace:` flux overrides the namespace component of each file, under this `flux-kustomization`. Thus, the `envoyfilter` wouldn't get deployed in `ngsa` namespace.
No envoyfilter == no burstheader